recipes/xvega/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "xvega" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "395d2c6890a4d9e28764ab39569663647138a7a9430f37c08fedd04c2cbd2dff" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/QuantStack/xvega/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
```

Comment:
This should have a run_exports

Reply:
xvega is header-only, so probably not at this point.

Reply:
Oh ok